pahfit/model.py

```diff
@@ -415,30 +424,222 @@ def plot(self, spec=None, redshift=None):
 
         redshift : float
             Redshift used to shift from the physical model, to the
-            observed model.
+            observed model. If None, it will be taken from spec.redshift
 
-            If None, will be taken from spec.redshift
+        use_instrument_fwhm : bool
+            For the lines, the default is to use the fwhm values
+            contained in the features table. When set to True, the fwhm
+            will be determined by the instrument model instead.
+
+        label_lines : bool
+            Add labels with the names of the lines, at the position of
+            each line.
```

Comment:
For now it is only for the lines, since this was something specific I needed. Should be easily expanded, for any feature that has the 'wavelength' parameter. I have a draft where it does this for the PAH bands, in a different, more minimalistic plotting function.

Reply:
Nice.  Put a pin in for later.